core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/DatabricksUtilities.scala

```diff
@@ -59,9 +59,22 @@ object DatabricksUtilities extends HasHttpClient {
   // Execution Params
   val TimeoutInMillis: Int = 40 * 60 * 1000
 
-  val NotebookFiles: Array[File] = Option(
-    FileUtilities.join(BuildInfo.baseDirectory.getParent, "notebooks").getCanonicalFile.listFiles()
-  ).get
+  val NotebookFiles: Array[File] = {
```

Comment:
nit: could remove mutability with recursion

https://stackoverflow.com/questions/2637643/how-do-i-list-all-files-in-a-subdirectory-in-scala

Reply:
Thanks! That's what I wanted >_<